frontend-react/src/components/Admin/AdminLastMileFailuresTable.tsx

```diff
@@ -9,24 +9,251 @@ import {
+ */
+const DataLoadRenderTable = (props: {
+    daysToShow: string;
+    filterText: string;
+    handleRetrySendClick: (jsonRowData: string) => void;
+    handleShowDetailsClick: (jsonRowData: string) => void;
+}) => {
     const lastMileData: AdmSendFailuresResource[] = useResource(
         AdmSendFailuresResource.list(),
-        { days_to_show: daysToShow }
+        { days_to_show: props.daysToShow }
     );
+    const lastMileResends: AdmAction[] = useResource(AdmAction.list(), {
+        days_to_show: props.daysToShow,
+    });
+
+    const fiterResends = (reportId: string) => {
+        return lastMileResends.filter((each) => each.filterMatch(reportId));
+    };
+
+    const rows = lastMileData
+        .filter((eachRow) => eachRow.filterMatch(props.filterText))
+        .map((eachRow) => {
+            // would be nice if org and receiver name were separate
+            const parts = eachRow.receiver.split(".") || eachRow.receiver;
+            const org = parts[0] || "";
+            const recvrName = parts.slice(1).join(".");
+            const linkRecvSettings = `/admin/orgreceiversettings/org/${org}/receiver/${recvrName}/action/edit`;
+            const resends = fiterResends(eachRow.reportId);
+            const dataForDialog: DataForDialog = {
+                info: eachRow,
+                resends: resends,
+            };
+            return (
+                <tr key={`lastmile_row_${eachRow.pk()}`}>
+                    <td>{formatDate(eachRow.failedAt)}</td>
+                    <td>
+                        <Button
+                            type="button"
+                            unstyled
+                            className={"font-mono-xs"}
+                            title={"Show Info"}
+                            key={`details_${eachRow.pk()}`}
+                            onClick={() =>
+                                props.handleShowDetailsClick(
+                                    JSON.stringify(dataForDialog, null, 4)
+                                )
+                            }
+                        >
+                            {eachRow.reportId}
+                            {" ⧉"}
+                        </Button>
+                        <span
+                            className={"rs-resendmarker"}
+                            title={"Resends attempted."}
+                        >
+                            {resends.length ? " ⚠️ " : null}
+                        </span>
+                    </td>
+                    <td>
+                        <Link
+                            title={"Jump to Settings"}
+                            to={linkRecvSettings}
+                            key={`recv_link_${eachRow.pk()}`}
+                            className={"font-mono-xs"}
+                        >
+                            {eachRow.receiver}
+                        </Link>
+                    </td>
+                    <td>
+                        <Button
```

Comment:
This can be achieved with the ActionableColumn feature.

Shape: https://github.com/CDCgov/prime-reportstream/blob/58599e355ad2a0b357005190c185a8b8497f0b81/frontend-react/src/components/Table/Table.tsx#L24-L27

Code: https://github.com/CDCgov/prime-reportstream/blob/58599e355ad2a0b357005190c185a8b8497f0b81/frontend-react/src/components/Table/ColumnData.tsx#L94-L114